lib/command/src/Obelisk/Command.hs

```diff
@@ -161,6 +161,8 @@ deployInitOpts = DeployInitOpts
   <$> strArgument (action "directory" <> metavar "DEPLOYDIR" <> help "Path to a directory that it will create")
   <*> strOption (long "ssh-key" <> action "file" <> metavar "SSHKEY" <> help "Path to an ssh key that it will symlink to")
   <*> some (strOption (long "hostname" <> metavar "HOSTNAME" <> help "hostname of the deployment target"))
+  <*> strOption (long "ssl-hostname" <> metavar "SSLHOSTNAME" <> help "SSL hostname")
```

Comment:
How about "The hostname that end-users will use to access the site"

Also, come to think of it, this should just be taken from the route, shouldn't it?

Reply:
@srid That sounds good.  Let's make sure we cover all our bases, e.g. if someone passes an invalid URL or a URL with a path, etc.

Reply:
Here's the full range of possible route validation:

```haskell
    InvalidRoute_MissingScheme uri -> route uri "must have an URI scheme"
    InvalidRoute_NotHttps uri -> route uri "must be HTTPS"
    InvalidRoute_MissingHost uri -> route uri "must contain a hostname"
    InvalidRoute_HasPort uri -> route uri "cannot specify port"
    InvalidRoute_HasPath uri -> route uri "cannot contain path"
 ```